aries-site/src/examples/templates/global-header/AppIdentityExample.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Box, CheckBox } from 'grommet';
+import { AppIdentity, UserContext } from '.';
+
+const defaultUser = {
+  firstName: 'John',
+  lastName: 'Doe',
+  email: 'john.doe@hpe.com',
+  image: '//s.gravatar.com/avatar/99020cae7ff399a4fbea19c0634f77c3?s=80',
+};
+
+export const AppIdentityExample = () => {
+  const [user, setUser] = useState(defaultUser);
+  const [checked, setChecked] = useState(true);
+  const [isAruba, setIsAruba] = useState();
+
+  const onToggle = () => {
+    if (user) setUser();
+    else setUser(defaultUser);
+
+    setChecked(!checked);
+  };
+
+  return (
+    <UserContext.Provider value={{ user, setUser }}>
+      <Box gap="small" align="start">
+        <AppIdentity title="Service Name" brand={!isAruba ? 'hpe' : 'aruba'} />
```

Comment:
State/user context seems like it is not accounting for Aruba=true signed in false.

![Screen Shot 2020-12-08 at 8 52 08 AM](https://user-images.githubusercontent.com/1756948/101508060-76d9cd00-3934-11eb-9e97-1b98eb3da5f5.png)

Reply:
This is desired, it should always say "Hewlett Packard Enterprise" in the case where a user is signed out. Maybe for clarity, if a user is signed out, I will not show the "Aruba Service" toggle.

Reply:
Updated.

Reply:
I like this approach. As I got further into the review, I started suspecting as much.